PSICover/PSICover.cs

```diff
@@ -163,31 +166,91 @@ class Analyzer {
          var code = File.ReadAllLines (file);
          for (int i = 0; i < code.Length; i++)
             code[i] = code[i].Replace ('<', '\u00ab').Replace ('>', '\u00bb');
+         int hitCount = 0;
          foreach (var block in blocks) {
             bool hit = hits[block.Id] > 0;
-            string tag = $"<span class=\"{(hit ? "hit" : "unhit")}\">";
+            if (hit) hitCount++;
+            string hitStyles = hit ? "hit tooltip" : "unhit";
+            string tag = $"<span class=\"{hitStyles}\">";
+            // Another solution, But this does not provide indentation.
+            //if (block.ELine - block.SLine > 0) tag = tag.Replace (">", " style=\"white-space:pre-line\">");
+
+            // Break multiple lines and attach span element to each line.
+            var linesCount = block.ELine - block.SLine;
+            if (linesCount > 0) {
+               // Consider start and end line from the multiple lines.
+               for (int i = 0; i <= linesCount; i++) {
+                  int lineIndex = block.SLine + i;
+                  var lineLength = code[lineIndex].Length;
+                  var trimmedCode = code[lineIndex].Trim ();

```

Comment:
Trim will create a new string. All we have to do is to figure out the index of non white space character.